error-issue/test/issue_builder.test.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {IssueBuilder} from '../src/issue_builder';
+import {ErrorReport} from '../src/types';
+
+const FULL_ISSUE_BODY = `Details
```

Comment:
This is only used once, how about inlining it? I actually prefer that you put this in a .txt file and read that file in the test - it's pretty long

Reply:
Done.